homeworks/html-css/AllaSydoriak/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "homework",
```

Comment:
Consider also adding `"private": true` field.

Reply:
Ok, i'll pay attention to it, thanks